Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/SimpleShapeMonteCarloAbsorptionTest.py

```diff
@@ -13,15 +13,16 @@
 
 
 class SimpleShapeMonteCarloAbsorptionTest(unittest.TestCase):
-    def setUp(self):
-        red_ws = Load('irs26176_graphite002_red.nxs')
-        red_ws = ConvertUnits(
-            InputWorkspace=red_ws,
+    @classmethod
+    def setUpClass(self):
+        __red_ws = Load('irs26176_graphite002_red.nxs')
```

Comment:
Is there a reason were using anonymous naming here, this is usually indicative of a code smell?

Reply:
Nope I'll fix that